pkg/types/installconfig.go

```diff
@@ -96,7 +96,10 @@ type Networking struct {
 	// Type is the network type to install
 	Type netopv1.NetworkType `json:"type"`
 
-	// ServiceCIDR is the ip block from which to assign service IPs
+	// MachineCIDR is the IP address space from which to assign machine IPs.
```

Comment:
can we move it above `Type` trying to keep the kubenetes specific options together.. ?

Reply:
> can we move it above `Type` trying to keep the kubenetes specific options together.. ?

I had it below `Type` to keep the CIDR properties together ;).  But sure, moved above `Type` with f27c67b -> 7aee83e.